src/test/java/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,180 @@
+package hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Exercise1 {
+
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5000, TimeUnit.MILLISECONDS);
+    }
+
+    @Test
+    public void browserTitleTest() {
+        SoftAssert sa = new SoftAssert();
+
+        // 1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        String actualTitle = driver.getTitle();
+        String expectedTitle = "Home Page";
+
+        // 2. Assert Browser title
+        sa.assertEquals(actualTitle, expectedTitle);
+        sa.assertAll();
+    }
+
+    @Test
+    public void userLoginTest() {
+        SoftAssert sa = new SoftAssert();
+
+        // 3. Perform login
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.cssSelector("#name")).sendKeys("Roman");
+        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        WebElement loginUserName = driver.findElement(By.id("user-name"));
+
+        // 4. Assert Username is loggined
+        sa.assertEquals(loginUserName.getText(), "ROMAN IOVLEV");
+        sa.assertAll();
+    }
+
+    //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+    @Test
+    public void headerSectionItemsTest() {
+        SoftAssert sa = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        List<WebElement> headerItems = driver.findElements(By
+                        .xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']/li"));
```

Comment:
Why do you decide use xpath here? I suppose that this locator could be changed to css and simplified